src/Carbon/Lang/fr.php

```diff
@@ -102,7 +102,7 @@
             case 'd':
                 return $number.($number === 1 ? 'er' : 'e');
 
-            // Words with feminine grammatical gender: semaine
+                // Words with feminine grammatical gender: semaine
```

Comment:
```suggestion
            // Words with feminine grammatical gender: semaine
```

Reply:
Those are actually `php-cs-fixer` (`vendor/bin/php-cs-fixer fix`) fixes, if I revert those back, [the CS](https://github.com/briannesbitt/Carbon/blob/master/.github/workflows/phpcs.yml) workflow will fail.

In this case, we need to negate the `statement_indentation` rule

Reply:
cf. https://github.com/briannesbitt/Carbon/pull/2851/commits/3205687d0c7c6d1db5ce726942a8c7fe9c20ee0c

Reply:
I believe some changes might be related to an update of version of PHP-CS-Fixer